LRG-Operations/Official/Testing/EU2/logs/configstore_log.txt


[2020-04-30 17:02:15] Failed to read store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf.tmp'
[2020-04-30 17:02:17] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-04-30 17:02:18] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-04-30 18:31:13] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-04-30 18:31:14] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-04-30 18:31:15] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-04-30 18:34:43] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-04-30 23:36:16] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-04-30 23:36:18] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-04-30 23:36:19] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-04-30 23:40:22] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-04-30 23:40:24] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-01 18:03:41] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-01 18:03:44] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-08 19:49:04] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-08 19:49:06] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-08 19:49:06] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-08 21:56:30] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-08 22:21:50] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-08 22:21:52] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-08 22:21:53] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 14:52:37] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 14:52:39] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 14:52:40] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 15:00:16] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 15:00:18] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 15:42:24] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 15:42:26] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 16:10:11] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 16:10:12] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 16:10:13] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 16:15:21] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 16:35:59] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 16:36:01] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 16:36:02] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 18:01:13] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 18:01:15] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 18:01:16] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 19:42:42] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 19:42:44] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 20:12:44] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 20:20:42] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 20:20:44] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 20:20:45] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 20:26:19] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-15 20:36:07] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-15 20:36:08] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-15 20:36:09] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-18 17:39:00] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-18 17:39:02] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-18 18:14:45] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-18 18:14:47] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-18 18:14:48] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-18 18:22:54] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-18 18:22:56] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-18 18:22:57] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-18 22:04:53] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-18 22:04:55] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-20 21:49:30] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-20 21:49:33] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-20 21:49:33] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-22 18:01:01] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-22 18:01:04] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-30 17:05:55] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-30 17:05:57] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-30 17:05:58] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-30 17:20:44] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-30 17:20:46] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-30 17:21:32] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-30 17:21:34] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-30 17:21:34] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-05-30 18:00:52] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-05-30 18:00:53] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-05-30 18:00:54] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-03 17:53:56] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-03 17:53:58] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 17:53:59] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 17:57:05] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-03 18:00:11] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-03 18:00:13] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 18:00:14] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-03 18:05:53] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-03 18:05:56] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 19:02:38] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 19:03:18] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 19:03:59] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 19:05:03] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-03 19:09:53] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-03 19:09:55] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 19:09:56] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-03 19:11:09] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-03 19:11:11] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-03 19:11:12] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-04 18:20:01] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-04 18:20:03] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-04 18:20:04] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-04 18:38:12] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-04 18:38:12] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-04 20:38:50] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-05 12:45:38] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-05 12:45:41] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-05 12:57:28] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-05 12:57:29] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-05 12:57:30] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-05 13:05:21] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-05 13:05:23] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-05 13:05:24] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-05 13:12:46] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-05 13:12:48] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-05 13:12:49] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-05 13:51:25] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-05 18:00:57] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-05 18:00:59] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-05 18:01:00] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-05 18:53:03] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-05 18:53:04] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-05 18:53:05] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-06 11:09:36] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-06 11:09:38] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-06 11:09:38] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-20 13:32:16] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-20 13:32:18] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-20 13:32:18] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-21 18:04:56] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-21 18:04:58] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-21 21:45:05] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-06-26 18:00:55] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-06-26 18:00:56] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-06-26 18:00:57] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-07-09 18:00:47] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-07-09 18:00:49] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-07-09 18:00:50] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-07-09 19:06:19] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-07-09 19:06:21] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-07-09 19:06:21] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-07-09 20:38:48] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-07-09 20:38:55] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-07-09 21:37:51] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-07-09 21:37:53] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-07-31 18:00:59] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-07-31 18:01:00] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-07-31 18:01:01] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-08-28 14:19:21] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-08-28 14:19:23] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-08-28 18:00:36] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-08-28 18:00:38] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-09-02 18:42:40] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-09-02 18:42:42] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'
[2020-09-02 18:42:42] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'


[2020-09-02 18:47:39] Loaded store 'install' from 'C:\Servers\Arma 3\EU2\config\config.vdf' successfully
[2020-09-02 18:47:41] Flushed store 'install' to 'C:\Servers\Arma 3\EU2\config\config.vdf'

LRG-Mike-Force/Mike Force v0.66.02/vn_mf_latest.vn_khe_sanh/config/params.hpp
class wipe_save
{
    title = $STR_vn_mf_param_wipe_save;
    values[] = {0, 1};
    texts[] = {"False", "True"};
    default = 0;
};

class wipe_save_desc
{
    title = $STR_vn_mf_param_wipe_save_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer0
{
    title = "";
    values[] = {""};
    texts[] = {""};
    default = "";
};

class enable_ranks
{
    title = $STR_vn_mf_param_enable_ranks;
    values[] = {0, 1};
    texts[] = {"False", "True"};
    default = 0;
};

class enable_ranks_desc
{
    title = $STR_vn_mf_param_enable_ranks_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer21 : Spacer0 {};

class allow_map_markers
{
    title = $STR_vn_mf_param_allow_map_markers;
    values[] = {0, 1};
    texts[] = {"False", "True"};
    default = 1;
};

class allow_map_markers_desc
{
    title = $STR_vn_mf_param_allow_map_markers_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer1 : Spacer0 {};

class hard_ai_limit
{
    title = $STR_vn_mf_param_hard_ai_limit;
    values[] = {60, 80, 100, 120, 140, 160, 180, 200};
    texts[] = {"60", "80 (recommended)", "100", "120", "140", "160", "180", "200"};
    default = 80;
};

class hard_ai_limit_desc
{
    title = $STR_vn_mf_param_hard_ai_limit_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer1_0 : Spacer0 {};

// gamemode specific start
class buildables_require_vehicles
{
    title = $STR_vn_mf_buildables_require_vehicles;
    values[] = {0, 1};
    texts[] = {"False", "True"};
    default = 0;
};

class buildables_require_vehicles_desc
{
    title = $STR_vn_mf_buildables_require_vehicles_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer2 : Spacer1 {};

class dawn_length
{
    title = $STR_vn_mf_dawn_length;
    values[] = {600, 1200, 1800, 2400, 3600, 5400, 7200, 9000, 10800};
    texts[] = {"10 minutes", "20 minutes", "30 minutes", "40 minutes", "1 hour", "1.5 hours", "2 hours", "2.5 hours", "3 hours"};
    default = 1200;
};

class Spacer3 : Spacer1 {};

class day_length
{
    title = $STR_vn_mf_day_length;
    values[] = {3600, 5400, 7200, 9000, 10800, 21600, 43200, 86400, 172800};
    texts[] = {"1 hour", "1.5 hours", "2 hours", "2.5 hours", "3 hours", "6 hours", "12 hours", "24 hours", "48 hours"};
    default = 7200;
};

class Spacer4 : Spacer1 {};

class dusk_length
{
    title = $STR_vn_mf_dusk_length;
    values[] = {600, 1200, 1800, 2400, 3600, 5400, 7200, 9000, 10800};
    texts[] = {"10 minutes", "20 minutes", "30 minutes", "40 minutes", "1 hour", "1.5 hours", "2 hours", "2.5 hours", "3 hours"};
    default = 1200;
};

class Spacer5 : Spacer1 {};

class night_length
{
    title = $STR_vn_mf_night_length;
    values[] = {600, 1200, 1800, 2400, 3600, 5400, 7200, 9000, 10800, 21600, 43200, 86400, 172800};
    texts[] = {"10 minutes", "20 minutes", "30 minutes", "40 minutes", "1 hour", "1.5 hours", "2 hours", "2.5 hours", "3 hours", "6 hours", "12 hours", "24 hours", "48 hours"};
    default = 1200;
};

class Spacer7 : Spacer1 {};

class building_sandbag_value
{
    title = $STR_vn_mf_building_sandbag_value;
    values[] = {10};
    texts[] = {"Default (10)"};
    default = 10;
};

class building_sandbag_value_desc
{
    title = $STR_vn_mf_building_sandbag_value_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer8 : Spacer1 {};

class advanced_revive
{
    title = $STR_vn_mf_advanced_revive;
    values[] = {1, 0};
    texts[] = {"True (Default)", "False"};
    default = 1;
};

class advanced_revive_desc
{
    title = $STR_vn_mf_advanced_revive_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer9 : Spacer1 {};

class snake_header
{
    title = $STR_vn_mf_param_snake_header;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer11 : Spacer1 {};

class snake_bite_chance
{
    title = $STR_vn_mf_param_snake_bite_chance;
    values[] = {0, 15, 25, 35, 50, 65, 75};
    texts[] = {"0% (Off)", "15%", "25%", "35%", "50% (Default)", "65%", "75%"};
    default = 50;
};

class snake_bite_chance_desc
{
    title = $STR_vn_mf_param_snake_bite_chance_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class snake_bite_distance
{
    title = $STR_vn_mf_param_snake_bite_distance;
    values[] = {25, 50, 75, 100, 125, 150};
    texts[] = {"25cm", "50cm", "75cm", "1m (Default)", "1.25m", "1.5m"};
    default = 100;
};

class snake_bite_distance_desc
{
    title = $STR_vn_mf_param_snake_bite_distance_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class snake_bite_frequency
{
    title = $STR_vn_mf_param_snake_bite_frequency;
    values[] = {150, 300, 450, 600};
    texts[] = {"2.5 minutes", "5 minutes (Default)", "7.5 minutes", "10 minutes"};
    default = 300;
};

class snake_bite_frequency_desc
{
    title = $STR_vn_mf_param_snake_bite_frequency_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class snake_bite_extra_time
{
    title = $STR_vn_mf_param_snake_bite_extra_time;
    values[] = {150, 300, 450, 600};
    texts[] = {"2.5 minutes", "5 minutes (Default)", "7.5 minutes", "10 minutes"};
    default = 300;
};

class snake_bite_extra_time_desc
{
    title = $STR_vn_mf_param_snake_bite_extra_time_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer12 : Spacer1 {};

class teams_header
{
    title = $STR_vn_mf_param_teams_header;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer13 : Spacer1 {};

class max_players_acav
{
    title = $STR_vn_mf_max_players_acav;
    values[] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 35, 40, 45, 50, 99};
    texts[] = {"0 players", "1 player", "2 players", "3 players", "4 players", "5 players", "6 players", "7 players", "8 players", "9 players", "10 players", "15 players", "20 players", "25 players", "30 players", "35 players", "40 players", "45 players", "50 players", "Default (99 players)"};
    default = 99;
};

class max_players_greenhornets
{
    title = $STR_vn_mf_max_players_greenhornets;
    values[] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 35, 40, 45, 50, 99};
    texts[] = {"0 players", "1 player", "2 players", "3 players", "4 players", "5 players", "6 players", "7 players", "8 players", "9 players", "10 players", "15 players", "20 players", "25 players", "30 players", "35 players", "40 players", "45 players", "50 players", "Default (99 players)"};
    default = 99;
};

class max_players_mikeforce
{
    title = $STR_vn_mf_max_players_mikeforce;
    values[] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 35, 40, 45, 50, 99};
    texts[] = {"0 players", "1 player", "2 players", "3 players", "4 players", "5 players", "6 players", "7 players", "8 players", "9 players", "10 players", "15 players", "20 players", "25 players", "30 players", "35 players", "40 players", "45 players", "50 players", "Default (99 players)"};
    default = 99;
};

class max_players_spiketeam
{
    title = $STR_vn_mf_max_players_spiketeam;
    values[] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 35, 40, 45, 50, 99};
    texts[] = {"0 players", "1 player", "2 players", "3 players", "4 players", "5 players", "6 players", "7 players", "8 players", "9 players", "10 players", "15 players", "20 players", "25 players", "30 players", "35 players", "40 players", "45 players", "50 players", "Default (99 players)"};
    default = 99;
};


class Spacer14 : Spacer1 {};

class enable_air_support
{
    title = $STR_vn_mf_enable_air_support;
    values[] = {0, 1, 2};
    texts[] = {"Off", "On (Default)", "Unique Supports Only"};
    default = 1;
};

class enable_air_support_desc
{
    title = $STR_vn_mf_enable_air_support_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};
  
  class enable_arty_support
{
    title = $STR_vn_mf_enable_arty_support;
    values[] = {0, 1};
    texts[] = {"Off", "On (Default)"};
    default = 1;
};

class Spacer15 : Spacer1 {};

class medical_header
{
    title = $STR_vn_mf_param_medical_header;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class Spacer16 : Spacer1 {};

class headshot_kill
{
    title = $STR_vn_mf_param_headshot_kill;
    values[] = {0, 1};
    texts[] = {"Off (Default)", "On"};
    default = 0;
};

class headshot_kill_desc
{
    title = $STR_vn_mf_param_headshot_kill_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};
class bleedout_time
{
    title = $STR_vn_mf_param_bleedout_time;
    values[] = {100, 200, 300, 400, 500};
    texts[] = {"100", "200", "300 (Default)", "400", "500"};
    default = 300;
};

class bleedout_time_desc
{
    title = $STR_vn_mf_param_bleedout_time_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class always_allow_withstand
{
    title = $STR_vn_mf_always_allow_withstand;
    values[] = {0, 1};
    texts[] = {"Off", "On (Default)"};
    default = 1;
};


class always_allow_withstand_desc
{
    title = $STR_vn_mf_always_allow_withstand_desc;

    values[] = {""};
    texts[] = {""};
    default = "";
};

class withstand_percentage
{
    title = $STR_vn_mf_param_withstand_percentage;
    values[] = {50, 70, 80, 100};
    texts[] = {"50%", "30%", "20% (Default)", "0%"};
    default = 80;
};

class withstand_percentage_desc
{
    title = $STR_vn_mf_param_withstand_percentage_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class remove_bandage_item
{
    title = $STR_vn_mf_param_remove_bandage_item;
    values[] = {0, 1};
    texts[] = {"Off", "On (Default)"};
    default = 1;
};


class remove_bandage_item_desc
{
    title = $STR_vn_mf_param_remove_bandage_item_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class remove_revive_item
{
    title = $STR_vn_mf_param_remove_revive_item;
    values[] = {0, 1};
    texts[] = {"Off (Default)", "On"};
    default = 0;
};

class remove_revive_item_desc
{
    title = $STR_vn_mf_param_remove_revive_item_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class revive_requirement
{
    title = $STR_vn_mf_param_revive_requirement;
    values[] = {0, 1};
    texts[] = {"Off (Default)", "On"};
    default = 0;
};

class respawn_delay
{
    title = $STR_vn_mf_param_respawn_delay;
    values[] = {5, 10, 20, 30};
    texts[] = {"5 seconds", "10 seconds", "20 seconds (Default)", "30 seconds"};
    default = 20;
};
  
class Spacer17 : Spacer1 {};

class hunger_loss_rate
{
    title = $STR_vn_mf_param_hunger_loss_rate;
    values[] = {1, 5, 10, 25, 50, 100};
    texts[] = {"0.01%", "0.05% (Default)", "0.1%", "0.25%", "0.5%", "1%"};
    default = 5;
};

class hunger_loss_rate_desc
{
    title = $STR_vn_mf_param_hunger_loss_rate_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};

class thirst_loss_rate
{
    title = $STR_vn_mf_param_thirst_loss_rate;
    values[] = {1, 5, 10, 25, 50, 100};
    texts[] = {"0.01%", "0.05%", "0.1% (Default)", "0.25%", "0.5%", "1%"};
    default = 10;
};

class thirst_loss_rate_desc
{
    title = $STR_vn_mf_param_thirst_loss_rate_desc;
    values[] = {""};
    texts[] = {""};
    default = "";
};
  
/*
class Spacer99: Spacer1 {};

class ArsenalSettings
{
    title = $STR_VN_PARAMS_ARSENAL_SETTINGS;
    values[] = {""};
    texts[] = {""};
    default = "";
};
class vn_whitelisted_arsenal_params_enable
{
    title = $STR_VN_PARAMS_ARSENAL_TOGGLE;
    values[] = {0, 1};
    texts[] = {$STR_VN_MISSIONS_DISABLED, $STR_VN_MISSIONS_ENABLED};
    default = 0; // TODO change to 0 in release
};
class vn_whitelisted_arsenal_params_scope
{
    title = $STR_VN_PARAMS_ARSENAL_SCOPE;
    values[] = {0, 1, 2};
    texts[] = {$STR_VN_PARAMS_ARSENAL_SCOPE_DLC, $STR_VN_PARAMS_ARSENAL_SCOPE_CUSTOM, $STR_VN_PARAMS_ARSENAL_SCOPE_UNSUNG};
    default = 0;
};
class vn_whitelisted_arsenal_params_rank
{
    title = $STR_VN_PARAMS_ARSENAL_RANK;
    values[] = {0, 1};
    texts[] = {$STR_VN_MISSIONS_ENABLED, $STR_VN_MISSIONS_DISABLED};
    default = 1;
};
class vn_whitelisted_arsenal_params_side
{
    title = $STR_VN_PARAMS_ARSENAL_SIDE;
    values[] = {0, 1};
    texts[] = {$STR_VN_PARAMS_ARSENAL_SIDE_RESTRICTED, $STR_VN_PARAMS_ARSENAL_UNRESTRICTED};
    default = 1; // TODO change to 0 in release
};
class vn_whitelisted_arsenal_params_equipment
{
    title = $STR_VN_PARAMS_ARSENAL_EQUIPMENT;
    values[] = {0, 1, 2, 3};
    texts[] = {$STR_VN_PARAMS_ARSENAL_EQUIPMENT_EQUIPMENT, $STR_VN_PARAMS_ARSENAL_EQUIPMENT_VEHICLES, $STR_VN_PARAMS_ARSENAL_EQUIPMENT_BOTH, $STR_VN_PARAMS_ARSENAL_EQUIPMENT_NONE};
    default = 0;
};
*/
